package.json

```diff
@@ -9,6 +9,7 @@
     "lint": "eslint -c .eslintrc.json --max-warnings 0 'src/**/*.ts{,x}'",
     "lint-fix": "yarn lint --fix",
     "start": "react-scripts start",
+    "start-https": "HTTPS=true react-scripts start",
```

Comment:
Why not make this the default `start` script?

Reply:
Clicking through all the certificate warnings can be very annoying for a user but also during rapid development. I think the easy part should be easy and if someone really needs ledger support locally, this is how they get it.